pagopa/io-app
# chore: [LLK-60] Key Generation info removed in favour of direct analytics

## Short description
This PR removes KeyGenerationInfo and sends the analytic event directly.

## List of changes proposed in this pull request
- `ts/features/lollipop/saga/index`: replaced KeyGenerationInfo storing by sending analytics event directly. Merged with `ts/sagas/startup/generateCryptoKeyPair`
- `ts/sagas/statup`: removed KeyInfoGeneration analytics sending, since it is done on the saga above
- `ts/features/lollipop/hooks/useLolipopLoginSource`: added KeyGenerationInfo analytics event on key regeneration
- `ts/features/lolipop/utils/crypto`: moved from ts/utils/crypto and stripped of any method that had only one file usage (such methods have been moved in consumers, like `useLollipopLoginSource`)

## How to test
Run the application (either dev-server or production are fine) and check that both successful and failure key generation events are properly tracked.
